modules/extract/functions/EXTRACT/fn_EXTRACT_CheckArrayInVeh.sqf

```diff
@@ -0,0 +1,13 @@
+#include "script_component.hpp"
+
+private _veh = _this select 0;
```

Comment:
use `params` instead of `_this select`

```sqf
params ["_veh", "_units"];
```

You can also add parameter limitations and defaults to reduce errors:

```sqf
params [
    ["_veh", objNull, [objNull]], 
    ["_units", [], [[]]]
];
```